src/darker/command_line.py

```diff
@@ -27,6 +27,9 @@ def parse_command_line(argv: List[str]) -> Namespace:
     isort_help = ["Also sort imports using the `isort` package"]
     if not isort:
         isort_help.append(f". {ISORT_INSTRUCTION} to enable usage of this option.")
+    parser.add_argument(
+        "--diff", action="store_true", help="print diff instead of modifying the file",
```

Comment:
All the other command help texts start with a title case word.
```suggestion
        "--diff", action="store_true", help="Print diff instead of modifying the file",
```

Reply:
Actually, let's make the text identical to that in `black --help`